database/migrations/000026_hierarchy.down.sql

```diff
@@ -0,0 +1,13 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
```

Comment:
is the down migration empty by design because there's no reasonable way to migrate down?

Reply:
I'm sorry there's a separate patch for that it seems - I was reading the patches one-by-one

Reply:
We squash the patches in the end before merging

Reply:
yeah that's fine I just didn't see the code because I was reading patch 1/6..